app/models/messages.py

```diff
@@ -0,0 +1,16 @@
+from app.models import *
+
+
+class Messages(Base, UserMixin):

```

Comment:
Take a look how the inheritance in SQLAlchemy is implement. You can apply it here for messages and user maybe 🤔